已是最新预发布版本但仍提示更新

<!-- 如果搜索过但未找到，请将 `[ ]` 替换为 `[x]` -->

- [x] 你是否在现有 [Issue列表](/docmirror/dev-sidecar/issues) 中搜索过相同问题，但未找到？

### Ⅰ. 请说明操作系统及DS的版本号：

1. 操作系统：Linux kali 6.18.12+kali-amd64 #1 SMP PREEMPT_DYNAMIC Kali 6.18.12-1kali1 (2026-02-25) x86_64 GNU/Linux
2. DS版本号：2.0.0.3

### Ⅱ. 问题描述：
我关闭了“忽略预览版”，然后升级了2.0.0.3，但每次开启dev-sidecar以后，还会弹出更新，且已是最新版本
### Ⅲ. 期望的结果：
更新后不再弹出更新
### Ⅳ. 如何复现问题？

1. 打开dev-sidecar（不是最新预览版） -> 设置 - > 将”忽略预发布版本“设置成“不忽略”
2. 重新打开dev-sidecar
3. 按照指示更新
4. 使用dpkg -i安装
5. 重新打开dev-sidecar
6. 已是最新版本，但弹出更新提示

### Ⅴ. 请提供相关的错误日志，尽可能的详细：（日志文件在 `${user.home}/.dev-sidecar/logs/` 目录下）

<details>
<summary>点击查看日志</summary>

```log
[2026-03-14T01:46:23.510] [INFO] server - 不拦截请求，直连目标服务器: registry.npmjs.org:443, headers: {"host":"registry.npmjs.org:443","proxy-connection":"close"}
[2026-03-14T01:46:56.494] [INFO] server - matchHostname: matched intercepts: 'safebrowsing.googleapis.com' -> { "^(?!.*(translate-pa).google(?:apis|usercontent)?.com).*google(?:apis|usercontent)?.com$": {".*":{"sni":"www.google.cn","desc":"部分需要走其它代理的服务"}} }
[2026-03-14T01:46:56.494] [INFO] server - getServerPromise, hostname: safebrowsing.googleapis.com:443, ssl: true, protocol: https
[2026-03-14T01:46:56.494] [INFO] server - Load fakeServerPromise from cache, hostname: safebrowsing.googleapis.com:443, ssl: true, serverPromiseObj: {"ssl":true,"port":443,"mappingHostNames":["*.googleapis.com","googleapis.com"]}
[2026-03-14T01:46:56.494] [INFO] server - ----- fakeServer connect: 127.0.0.1:41175 ➜ safebrowsing.googleapis.com:443 -----
[2026-03-14T01:46:56.494] [INFO] server - Proxy connect start: 127.0.0.1:41175
[2026-03-14T01:46:56.498] [INFO] server - fakeServer SNICallback: safebrowsing.googleapis.com:443
[2026-03-14T01:46:56.555] [INFO] server - matchHostname-all: get interceptOpts: 'safebrowsing.googleapis.com': {".*":{"sni":"www.google.cn","desc":"部分需要走其它代理的服务"}}
[2026-03-14T01:46:56.556] [ERROR] server - Request error: GET ➜ https://safebrowsing.googleapis.com:443/v4/threatListUpdates:fetch?$ct=application/x-protobuf&key=AIzaSyD3uzXks34szqk9WhKoFZypVPgdDbT3uPw&$httpMethod=POST&$req=ChUKE25hdmNsaWVudC1hdXRvLWZmb3gaJwgFEAIaGwoNCAUQBhgBIgMwMDEwARD1-R4aAhgEiGRa3CICIAIoARonCAEQAhobCg0IARAGGAEiAzAwMTABEMufFBoCGARyd6T5IgIgAigBGicIAxACGhsKDQgDEAYYASIDMDAxMAEQy5gUGgIYBMgQaOkiAiACKAEaJwgHEAIaGwoNCAcQBhgBIgMwMDEwARDvkRQaAhgEgnOL-CICIAIoARolCAkQAhoZCg0ICRAGGAEiAzAwMTABECQaAhgE2QsWPSICIAIoAQ==, error: TypeError: Cannot read properties of undefined (reading 'rejectUnauthorized')
    at Object.requestIntercept (/opt/dev-sidecar/resources/app.asar/mitmproxy.js:2:1218454)
    at e.requestIntercept.c.requestIntercept (/opt/dev-sidecar/resources/app.asar/mitmproxy.js:2:1277721)
    at k.host (/opt/dev-sidecar/resources/app.asar/mitmproxy.js:2:1255016)
    at new Promise (<anonymous>)
    at /opt/dev-sidecar/resources/app.asar/mitmproxy.js:2:1254837
    at e.exports.requestHandler (/opt/dev-sidecar/resources/app.asar/mitmproxy.js:2:1258712)
    at Server.<anonymous> (/opt/dev-sidecar/resources/app.asar/mitmproxy.js:2:1265792)
    at Server.emit (node:events:526:28)
    at parserOnIncoming (node:_http_server:951:12)
    at HTTPParser.parserOnHeadersComplete (node:_http_common:128:17)
[2026-03-14T01:47:00.350] [INFO] server - 不拦截请求，直连目标服务器: filters.adtidy.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"filters.adtidy.org:443"}
[2026-03-14T01:47:00.351] [INFO] server - 不拦截请求，直连目标服务器: filters.adtidy.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"filters.adtidy.org:443"}
[2026-03-14T01:47:00.352] [INFO] server - 不拦截请求，直连目标服务器: filters.adtidy.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"filters.adtidy.org:443"}
[2026-03-14T01:47:00.352] [INFO] server - 不拦截请求，直连目标服务器: filters.adtidy.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"filters.adtidy.org:443"}
[2026-03-14T01:47:00.353] [INFO] server - 不拦截请求，直连目标服务器: filters.adtidy.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"filters.adtidy.org:443"}
[2026-03-14T01:47:00.353] [INFO] server - 不拦截请求，直连目标服务器: filters.adtidy.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"filters.adtidy.org:443"}
[2026-03-14T01:47:57.102] [INFO] server - 不拦截请求，直连目标服务器: blacklist.tampermonkey.net:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"blacklist.tampermonkey.net:443"}
[2026-03-14T01:47:57.370] [ERROR] server - 直连失败: blacklist.tampermonkey.net:443, cost: 267 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -104,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2026-03-14T01:47:57.377] [INFO] server - 不拦截请求，直连目标服务器: blacklist.tampermonkey.net:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"blacklist.tampermonkey.net:443"}
[2026-03-14T01:47:57.617] [ERROR] server - 直连失败: blacklist.tampermonkey.net:443, cost: 240 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -104,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2026-03-14T01:49:14.523] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:49:14.524] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:49:14.702] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:49:14.703] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:49:14.928] [INFO] server - 不拦截请求，直连目标服务器: ynuf.aliapp.org:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"ynuf.aliapp.org:443"}
[2026-03-14T01:49:14.953] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:05.920] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:06.343] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:06.351] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:07.796] [INFO] server - 不拦截请求，直连目标服务器: account-cn-s.alibabacloud.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"account-cn-s.alibabacloud.com:443"}
[2026-03-14T01:51:13.737] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:13.740] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:13.741] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:13.741] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:13.742] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:13.744] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:19.790] [INFO] server - 不拦截请求，直连目标服务器: cws.alyasset.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"cws.alyasset.com:443"}
[2026-03-14T01:51:19.829] [INFO] server - 不拦截请求，直连目标服务器: oss.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"oss.aliyuncs.com:443"}
[2026-03-14T01:51:24.322] [ERROR] server - 直连失败: arms-retcode.aliyuncs.com:443, cost: 10580 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -104,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2026-03-14T01:51:24.436] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:24.443] [ERROR] server - 直连失败: arms-retcode.aliyuncs.com:443, cost: 10702 ms, errorMsg: read ECONNRESET
 Error: read ECONNRESET
    at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -104,
  code: 'ECONNRESET',
  syscall: 'read'
}
[2026-03-14T01:51:24.444] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:51:29.501] [INFO] server - 不拦截请求，直连目标服务器: webapi.amap.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"webapi.amap.com:443"}
[2026-03-14T01:51:29.501] [INFO] server - 不拦截请求，直连目标服务器: webapi.amap.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"webapi.amap.com:443"}
[2026-03-14T01:51:47.933] [INFO] server - 不拦截请求，直连目标服务器: emogine.insights.1688.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"emogine.insights.1688.com:443"}
[2026-03-14T01:51:47.974] [INFO] server - 不拦截请求，直连目标服务器: xux-web-config.oss-accelerate.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"xux-web-config.oss-accelerate.aliyuncs.com:443"}
[2026-03-14T01:51:51.799] [INFO] server - 不拦截请求，直连目标服务器: xstore.insights.1688.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"xstore.insights.1688.com:443"}
[2026-03-14T01:52:48.330] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
[2026-03-14T01:52:48.487] [INFO] server - 不拦截请求，直连目标服务器: arms-retcode.aliyuncs.com:443, headers: {"user-agent":"Mozilla/5.0 (X11; Linux x86_64; rv:140.0) Gecko/20100101 Firefox/140.0","proxy-connection":"keep-alive","connection":"keep-alive","host":"arms-retcode.aliyuncs.com:443"}
```
</details>

### Ⅵ. 有必要时，请提供 `${user.home}/.dev-sidecar/running.json` 文件内容：

<!-- 请将 'running.json' 文件的内容粘贴在这里，方便我们排查问题是否由配置错误导致。 -->

<details>
<summary>点击查看运行参数</summary>

```json
{
	"port": 12345,
	"intercepts": {
		"metaInfo": {
			"version": 202602231340,
			"updateLog": "更新ms-sso.copilot.microsoft.com规则,防止微软登录卡登录",
			"id": "official"
		},
		"github.com": {
			"^(/[^/]+){2,}/?(\\?.*)?$": {
				"tampermonkeyScript": "https://gitee.com/wangliang181230/dev-sidecar/raw/scripts/tampermonkey.js",
				"script": "https://gitee.com/wangliang181230/dev-sidecar/raw/scripts/GithubEnhanced-High-Speed-Download.user.js",
				"remark": "注：上面所使用的脚本地址，为高速镜像地址。",
				"desc": "油猴脚本：高速下载 Git Clone/SSH、Release、Raw、Code(ZIP) 等文件 (公益加速)、项目列表单文件快捷下载、添加 git clone 命令"
			},
			"^((/[^/]+){2,})/raw((/[^/]+)+\\.(jpg|jpeg|png|gif))(\\?.*)?$": {
				"cacheDays": 365,
				"desc": "仓库内图片重定向，缓存1年。"
			},
			"^((/[^/]+){2,})/raw((/[^/]+)+\\.js)(\\?.*)?$": {
				"responseReplace": {
					"headers": {
						"content-type": "application/javascript; charset=utf-8"
					}
				},
				"desc": "仓库内脚本，设置响应头Content-Type。作用：方便script拦截器直接使用，避免引起跨域问题和脚本内容限制问题。"
			},
			".*": {
				"sni": "baidu.com"
			},
			"/fluidicon.png": {
				"cacheDays": 365,
				"desc": "Github那只猫的图片，缓存1年"
			},
			"^(/[^/]+){2}/pull/\\d+/open_with_menu.*$": {
				"cacheDays": 7,
				"desc": "PR详情页：标题右边那个Code按钮的HTML代码请求地址，感觉上应该可以缓存。暂时先设置为缓存7天"
			}
		},
		"api.github.com": {
			".*": {
				"sni": "baidu.com"
			},
			"^/_private/browser/stats$": {
				"success": true,
				"desc": "github的访问速度分析上传，没有必要，直接返回成功"
			}
		},
		"github.githubassets.com": {
			"/assets/fakefile.js": {
				"abort": true,
				"cacheDays": 365
			},
			"^(/[^/]+)*/[^./]+\\.(svg|png|gif|jpg|jpeg|ico|js|css)(\\?.*)?$": {
				"cacheDays": 365,
				"desc": "图片、JS文件、CSS文件，缓存1年"
			},
			".*.backup": {
				"desc": "github.com域名下，该请求已经不存在，此配置暂时先备份掉。",
				"proxy": "github.com",
				"sni": "baidu.com",
				"responseReplace": {
					"headers": {
						"access-control-allow-origin": "*",
						"cross-origin-resource-policy": "cross-origin",
						"set-cookie": "[remove]"
					}
				}
			},
			".*": {
				"sni": "baidu.com"
			}
		},
		"opengraph.githubassets.com": {
			"^/(([^/]+/){3}issues/\\d+)?(\\?.*)?$": {
				"cacheDays": 365
			},
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.githubusercontent.com": {
			".*": {
				"sni": "baidu.com",
				"requestReplace": {
					"headers": {
						"accept-language": "en-US,en;q=0.8"
					}
				}
			}
		},
		"collector.github.com": {
			"/github/collect": {
				"success": true,
				"status": 204,
				"desc": "采集数据，快速成功"
			},
			".*": {
				"sni": "baidu.com"
			}
		},
		"gist.github.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.github.io": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.gravatar.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.windows.net": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"fonts.(googleapis|gstatic).com": {
			".*": {
				"proxy": "fonts.googleapis.cn"
			}
		},
		"^(?!.*(translate-pa).google(?:apis|usercontent)?.com).*google(?:apis|usercontent)?.com$": {
			".*": {
				"sni": "www.google.cn",
				"desc": "部分需要走其它代理的服务"
			}
		},
		"^.*(youtube.com|gstatic.com|youtube.nocookie.com|youtu.be|ggpht.com|i.ytimg.com|blogger.com|doodles.google|about.google|android.com)$": {
			".*": {
				"sni": "www.google.cn"
			}
		},
		"www.gstatic.com": {
			"/recaptcha/.*": {
				"proxy": "www.recaptcha.net"
			}
		},
		"ms-sso.copilot.microsoft.com": {
			".*": {
				"sni": "microsoft.com"
			}
		},
		"www.docker.com": {
			".*": {
				"sni": "www.docker.com"
			}
		},
		"login.docker.com": {
			".*": {
				"sni": "login.docker.com"
			},
			"/favicon.ico": {
				"proxy": "hub.docker.com",
				"sni": "baidu.com",
				"desc": "登录页面的ico，采用hub.docker.com的"
			}
		},
		"download.docker.com": {
			".*": {
				"sni": "download.docker.com"
			}
		},
		"hub.docker.com": {
			".*": {
				"sni": "none"
			}
		},
		"*.docker.com": {
			".*": {
				"sni": "www.docker.com"
			}
		},
		"*.pixiv.net": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.pixiv.org": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.pximg.net": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.ads-pixiv.net": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.nikke-global.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"i.pximg.net": {
			".*": {
				"cacheDays": 365,
				"requestReplace": {
					"headers": {
						"referer": "https://www.pixiv.net/"
					},
					"desc": "篡改请求头'Referer'，使Pixiv图片链接可以单独在浏览器打开"
				}
			}
		},
		"*.youtube.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.youtu.be": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.youtube-nocookie.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.ggpht.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"i.ytimg.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"cdn.jsdelivr.net": {
			"^/.*\\.(js|css|png|jpg|jpeg|gif|json)(\\?.*)?$": {
				"proxy": "fastly.jsdelivr.net",
				"backup": [
					"gcore.jsdelivr.net"
				]
			}
		},
		"*.greasyfork.org": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.cn-greasyfork.org": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.huggingface.co": {
			".*": {
				"sni": "huggingface.cn"
			}
		},
		"cn.vuejs.org": {
			".*": {
				"sni": "vuejs.org"
			}
		},
		"*z-library.sk": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*z-lib.help": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*.steamserver.net": {
			".*": {
				"sni": "www.baidu.com",
				"desc": "steam登录不稳定？无法复现先这么写"
			}
		},
		"images.steamusercontent.com": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*steamcommunity.com": {
			"^(?!/discussions|/Profile|/app.*/discussions).*$": {
				"sni": "www.baidu.com",
				"desc": "讨论区锁区,考虑不拦截丢给彩蛋"
			}
		},
		"^(?!.*cloudflare).*steamstatic.com$": {
			".*": {
				"sni": "baidu.com",
				"desc": "steam社区数据不能也不需调整sni,直接直连"
			}
		},
		"*.steampowered.com": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"external-content.duckduckgo.com": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*duckduckgo.com": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*onedrive.live.com": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*dropbox.com": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*f-droid.org": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"fdroid.org": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"*apkmirror.com": {
			".*": {
				"sni": "none"
			}
		},
		"jsd.proxy.aks.moe": {
			"^.*\\?DS_DOWNLOAD$": {
				"requestReplace": {
					"doDownload": true
				},
				"responseReplace": {
					"doDownload": true
				}
			}
		},
		"fastly.jsdelivr.net": {
			"^.*\\?DS_DOWNLOAD$": {
				"requestReplace": {
					"doDownload": true
				},
				"responseReplace": {
					"doDownload": true
				}
			}
		},
		"jsdelivr.pai233.top": {
			"^.*\\?DS_DOWNLOAD$": {
				"requestReplace": {
					"doDownload": true
				},
				"responseReplace": {
					"doDownload": true
				}
			}
		},
		"raw.incept.pw": {
			"^.*\\?DS_DOWNLOAD$": {
				"requestReplace": {
					"doDownload": true
				},
				"responseReplace": {
					"doDownload": true
				}
			}
		},
		"*.msecnd.net": {
			".*": {
				"sni": "baidu.com"
			}
		},
		"*.instagram.com": {
			".*": {
				"sni": "g.cn"
			}
		},
		"*.cdninstagram.com": {
			".*": {
				"sni": "g.cn"
			}
		},
		"*.intercom.io": {
			".*": {
				"sni": "g.cn"
			}
		},
		"*startpage.com": {
			".*": {
				"sni": "www.baidu.com"
			}
		},
		"camo.githubusercontent.com": {
			"^[a-zA-Z0-9/]+(\\?.*)?$": {
				"cacheDays": 365,
				"desc": "图片，缓存1年"
			}
		},
		"customer-stories-feed.github.com": {
			".*": {
				"proxy": "customer-stories-feed.fastgit.org"
			}
		},
		"user-images.githubusercontent.com": {
			"^/.*\\.png(\\?.*)?$": {
				"cacheDays": 365,
				"desc": "用户在PR或issue等内容中上传的图片，缓存1年。注：每张图片都有唯一的ID，不会重复，可以安心缓存"
			}
		},
		"private-user-images.githubusercontent.com": {
			"^/.*\\.png(\\?.*)?$": {
				"cacheDays": 30,
				"desc": "用户在PR或issue等内容中上传的图片，缓存30天"
			}
		},
		"avatars.githubusercontent.com": {
			"^/u/\\d+(\\?.*)?$": {
				"cacheDays": 365,
				"desc": "用户头像，缓存1年"
			}
		},
		"www.google.com": {
			"/recaptcha/.*": {
				"proxy": "www.recaptcha.net"
			}
		},
		"ajax.googleapis.com": {
			".*": {
				"proxy": "ajax.lug.ustc.edu.cn",
				"backup": [
					"gapis.geekzu.org"
				],
				"test": "ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"
			}
		},
		"fonts.googleapis.com": {
			".*": {
				"proxy": "fonts.loli.net",
				"test": "https://fonts.googleapis.com/css?family=Oswald"
			}
		},
		"themes.googleapis.com": {
			".*": {
				"proxy": "themes.loli.net",
				"backup": [
					"themes.proxy.ustclug.org"
				]
			}
		},
		"themes.googleusercontent.com": {
			".*": {
				"proxy": "google-themes.proxy.ustclug.org"
			}
		},
		"clients*.google.com": {
			".*": {
				"abort": false,
				"desc": "设置abort：true可以快速失败，节省时间"
			}
		},
		"www.googleapis.com": {
			".*": {
				"abort": false,
				"desc": "设置abort：true可以快速失败，节省时间"
			}
		},
		"lh*.googleusercontent.com": {
			".*": {
				"abort": false,
				"desc": "设置abort：true可以快速失败，节省时间"
			}
		},
		"*.s3.1amazonaws1.com": {
			"/sqlite3/.*": {
				"redirect": "npm.taobao.org/mirrors"
			}
		},
		"*.carbonads.com": {
			"/carbon.*": {
				"abort": true,
				"desc": "广告拦截"
			}
		},
		"*.buysellads.com": {
			"/ads/.*": {
				"abort": true,
				"desc": "广告拦截"
			}
		}
	},
	"preSetIpList": {
		"github.com": {
			"20.200.245.247": true,
			"20.27.177.113": true,
			"20.205.243.166": true,
			"20.26.156.215": false,
			"20.87.245.0": false,
			"4.237.22.38": false,
			"20.201.28.151": false,
			"140.82.113.3": false,
			"140.82.114.4": false,
			"140.82.116.3": false,
			"140.82.116.4": false,
			"140.82.121.3": false,
			"140.82.121.4": false
		},
		"gist.github.com": {
			"20.27.177.113": true,
			"20.200.245.247": true,
			"20.205.243.166": false,
			"140.82.116.3": true,
			"140.82.116.4": true,
			"4.237.22.38": true
		},
		"github.dev": {
			"20.43.185.14": true,
			"20.99.227.183": true,
			"51.137.3.17": true,
			"52.224.38.193": true
		},
		"hub.docker.com": {
			"144.34.172.106": false,
			"69.63.207.139": false,
			"47.236.233.96": true
		},
		"sessions-bugsnag.docker.com": {
			"54.156.14.194": true,
			"54.208.73.48": true,
			"100.29.167.200": true
		},
		"i.pximg.net": {
			"210.140.139.132": true,
			"210.140.139.137": true,
			"203.137.29.48": true,
			"210.140.139.134": true,
			"203.137.29.49": true,
			"210.140.139.133": true,
			"210.140.139.135": true,
			"203.137.29.47": true
		},
		"a.pixiv.org": {
			"210.140.139.182": true,
			"210.140.139.183": true,
			"210.140.139.184": true
		},
		"api.fanbox.cc": {
			"172.64.146.116": true
		},
		"*.fanbox.cc": {
			"210.140.139.155": true
		},
		"fonts.googleapis.cn": {
			"120.253.255.34": true,
			"220.181.174.98": true,
			"114.250.67.34": true,
			"203.208.41.98": true,
			"220.181.174.162": true
		},
		"^(aistudio|.*pa.clients6|apis|gemini|console.cloud).google.com$": {
			"8.137.102.117": {
				"desc": "AI Studio相关,阿里云,解锁地区限制,新加坡,仅大陆访问"
			},
			"47.102.115.14": false
		},
		"stitch.withgoogle.com": {
			"8.137.102.117": {
				"desc": "AI Studio相关,阿里云,解锁地区限制,新加坡,仅大陆访问"
			},
			"47.102.115.14": false
		},
		"scholar.googleusercontent.com": {
			"142.251.190.206": true,
			"172.217.204.206": true,
			"172.253.122.206": true
		},
		"(*account*|scholar).google.com": {
			"172.217.204.206": true
		},
		"^.*(with)?google(apis|usercontent)?.com.*$": {
			"8.137.102.117": {
				"desc": "阿里云中转"
			},
			"142.251.189.206": {
				"desc": "自动地区官方"
			}
		},
		"^.*(youtube.com|gstatic.com|youtube.nocookie.com|youtu.be|ggpht.com|i.ytimg.com|blogger.com|doodles.google|about.google|android.com|antigravity.google)$": {
			"8.137.102.117": {
				"desc": "阿里云中转"
			},
			"142.251.189.206": {
				"desc": "自动地区官方"
			}
		},
		"*.youtube.com": {
			"8.137.102.117": true
		},
		"*.youtu.be": {
			"8.137.102.117": true
		},
		"*.youtube-nocookie.com": {
			"8.137.102.117": true
		},
		"*.ggpht.com": {
			"8.137.102.117": true
		},
		"i.ytimg.com": {
			"8.137.102.117": true
		},
		"*.gstatic.com": {
			"8.137.102.117": true
		},
		"*.huggingface.co": {
			"3.167.200.113": true
		},
		"*.greasyfork.org": {
			"96.126.98.220": true,
			"50.116.4.196": true
		},
		"*.cn-greasyfork.org": {
			"96.126.98.220": true,
			"50.116.4.196": true
		},
		"*.instagram.com": {
			"163.70.159.174": true,
			"57.144.144.34": true,
			"57.144.216.34": true,
			"157.240.229.174": true,
			"57.144.188.34": true,
			"157.240.210.174": true,
			"157.240.252.174": true,
			"102.132.97.174": true,
			"31.13.94.174": true,
			"31.13.85.174": true,
			"102.132.104.174": true
		},
		"*.cdninstagram.com": {
			"57.144.152.192": true,
			"57.144.220.192": true,
			"57.144.216.192": true
		},
		"*z-library.sk": {
			"176.123.7.105": true
		},
		"*z-lib.help": {
			"176.123.7.105": true
		},
		"community.cloudflare.steamstatic.com": {
			"23.209.46.70": true,
			"104.18.42.105": true,
			"172.64.145.151": true
		},
		"*.steamstatic.com": {
			"151.101.91.52": true,
			"146.75.47.52": true,
			"146.75.115.52": true,
			"151.101.111.52": true,
			"151.101.3.52": true,
			"151.101.195.52": true,
			"151.101.79.52": true,
			"151.101.67.52": true,
			"146.75.51.52": true
		},
		"cdn.steamcommunity.com": {
			"23.221.227.5": true
		},
		"*.steamcommunity.com": {
			"184.50.187.66": true
		},
		"^(store|checkout).steampowered.com$": {
			"72.246.28.155": true,
			"2.19.100.78": true,
			"23.15.141.198": true,
			"23.206.181.48": true,
			"23.35.101.126": true
		},
		"external-content.duckduckgo.com": {
			"20.43.160.189": true
		},
		"*duckduckgo.com": {
			"20.43.161.105": true
		},
		"*onedrive.live.com": {
			"13.107.42.13": true
		},
		"*dropbox.com": {
			"162.125.248.18": true
		},
		"forum.f-droid.org": {
			"37.218.242.53": true
		},
		"*f-droid.org": {
			"37.218.243.72": true
		},
		"fdroid.org": {
			"37.218.243.72": true
		},
		"*apkmirror.com": {
			"104.17.67.215": true
		},
		"*startpage.com": {
			"67.63.58.139": true
		},
		"cdn.jsdelivr.net": {
			"104.16.89.20": true
		},
		"api.github.com": {
			"20.26.156.210": true,
			"20.27.177.116": true,
			"20.87.245.6": true,
			"20.200.245.245": true,
			"20.201.28.148": true,
			"20.205.243.168": true,
			"20.248.137.49": true,
			"140.82.112.5": true,
			"140.82.113.6": true,
			"140.82.116.6": true,
			"140.82.121.6": true
		},
		"codeload.github.com": {
			"20.26.156.216": true,
			"20.27.177.114": true,
			"20.87.245.7": true,
			"20.200.245.246": true,
			"20.201.28.149": true,
			"20.205.243.165": true,
			"20.248.137.55": true,
			"140.82.113.9": true,
			"140.82.114.10": true,
			"140.82.116.10": true,
			"140.82.121.9": true
		},
		"*.githubusercontent.com": {
			"146.75.92.133": true,
			"199.232.88.133": true,
			"199.232.144.133": true
		},
		"viewscreen.githubusercontent.com": {
			"140.82.112.21": true,
			"140.82.112.22": true,
			"140.82.113.21": true,
			"140.82.113.22": true,
			"140.82.114.21": true,
			"140.82.114.22": true
		},
		"github.io": {
			"185.199.108.153": true,
			"185.199.109.153": true,
			"185.199.110.153": true,
			"185.199.111.153": true
		},
		"*.githubassets.com": {
			"185.199.108.154": true,
			"185.199.109.154": true,
			"185.199.110.154": true,
			"185.199.111.154": true
		},
		"^(analytics|ghcc)\\.githubassets\\.com$": {
			"185.199.108.153": true,
			"185.199.110.153": true,
			"185.199.109.153": true,
			"185.199.111.153": true
		},
		"*.pixiv.net": {
			"210.140.139.154": true,
			"210.140.139.157": true,
			"210.140.139.160": true
		}
	},
	"dns": {
		"providers": {
			"cf-DoT": {
				"server": "tls://1.1.1.1",
				"sni": "baidu.com",
				"cacheSize": 1000
			},
			"cf-DoH": {
				"server": "https://cloudflare-dns.com/dns-query",
				"sni": "baidu.com",
				"cacheSize": 1000
			},
			"Google-DoH": {
				"server": "https://dns.google/dns-query",
				"sni": "www.google.cn",
				"cacheSize": 1000,
				"desc": "不可用"
			},
			"aliyun": {
				"type": "https",
				"server": "https://dns.alidns.com/dns-query",
				"cacheSize": 1000
			},
			"cloudflare": {
				"type": "https",
				"server": "https://1.1.1.1/dns-query",
				"cacheSize": 1000
			},
			"quad9": {
				"type": "https",
				"server": "https://9.9.9.9/dns-query",
				"cacheSize": 1000
			},
			"safe360": {
				"type": "https",
				"server": "https://doh.360.cn/dns-query",
				"cacheSize": 1000,
				"forSNI": true
			},
			"rubyfish": {
				"type": "https",
				"server": "https://rubyfish.cn/dns-query",
				"cacheSize": 1000
			}
		},
		"mapping": {
			"*.jetbrains.com": "cf-DoT",
			"*.azureedge.net": "cf-DoT",
			"*.stackoverflow.com": "cf-DoT",
			"*.github.com": "cf-DoT",
			"*github*.com": "cf-DoT",
			"*.github.io": "cf-DoT",
			"*.docker.com": "cf-DoT",
			"*.electronjs.org": "cf-DoT",
			"*.amazonaws.com": "cf-DoT",
			"*.yarnpkg.com": "cf-DoT",
			"*.cloudfront.net": "cf-DoT",
			"*.cloudflare.com": "cf-DoT",
			"img.shields.io": "cf-DoT",
			"*.vuepress.vuejs.org": "cf-DoT",
			"*.gh.docmirror.top": "cf-DoT",
			"*.v2ex.com": "cf-DoT",
			"*.pypi.org": "cf-DoT",
			"*.pixiv.org": "cf-DoT",
			"*.pximg.net": "cf-DoT",
			"*.onesignal.com": "cf-DoT",
			"*.iubenda.com": "cf-DoT"
		},
		"speedTest": {
			"interval": 60000,
			"hostnameList": [
				"google.com",
				"zh.wikipedia.org",
				"steamgames.com",
				"epicgames-download1-1251447533.file.myqcloud.com",
				"store.steampowered.com",
				"community.akamai.steamstatic.com",
				"cdn.akamai.steamstatic.com",
				"steamcdn-a.akamaihd.net",
				"store.akamai.steamstatic.com",
				"steamcommunity.com",
				"api.steampowered.com"
			],
			"dnsProviders": [
				"cf-DoT",
				"cf-DoH"
			],
			"enabled": true
		}
	},
	"whiteList": {
		"*.icloud.com": true,
		"*.lenovo.net": true,
		"localhost": true,
		"127.*.*.*": true,
		"192.168.*.*": true,
		"*.cn": true,
		"*china*": true,
		"*.dingtalk.com": true,
		"*.apple.com": true,
		"*.microsoft.com": true,
		"*.alipay.com": true,
		"*.qq.com": true,
		"*.baidu.com": true
	},
	"enabled": true,
	"host": "127.0.0.1",
	"setting": {
		"NODE_TLS_REJECT_UNAUTHORIZED": true,
		"verifySsl": true,
		"script": {
			"enabled": true,
			"defaultDir": "./extra/scripts/"
		},
		"userBasePath": "/home/zhaobokai/.dev-sidecar",
		"rootCaFile": {
			"certPath": "/home/zhaobokai/.dev-sidecar/dev-sidecar.ca.crt",
			"keyPath": "/home/zhaobokai/.dev-sidecar/dev-sidecar.ca.key.pem"
		},
		"defaultTimeout": 20000,
		"defaultKeepAliveTimeout": 30000,
		"timeoutMapping": {
			"github.com": {
				"timeout": 20000,
				"keepAliveTimeout": 30000
			}
		},
		"lowSpeedDelay": 200
	},
	"compatible": {
		"connect": {},
		"request": {}
	},
	"intercept": {
		"enabled": true
	},
	"app": {
		"autoStart": {
			"enabled": true
		},
		"theme": "light",
		"skipPreRelease": false,
		"closeStrategy": 1,
		"mode": "default",
		"remoteConfig": {
			"enabled": true,
			"url": "https://gitee.com/wangliang181230/dev-sidecar/raw/docmirror2.x/packages/core/src/config/remote_config.json",
			"personalUrl": ""
		},
		"startShowWindow": true,
		"needCheckHideWindow": true,
		"showHideShortcut": "Alt + S",
		"windowSize": {
			"width": 900,
			"height": 750
		},
		"autoChecked": true,
		"dock": {
			"hideWhenWinClose": false
		},
		"showShutdownTip": true,
		"logFileSavePath": "/home/zhaobokai/.dev-sidecar/logs",
		"keepLogFileCount": 15,
		"maxLogFileSize": 1,
		"maxLogFileSizeUnit": "GB"
	},
	"plugin": {
		"pip": {
			"setting": {
				"trustedHost": "mirrors.aliyun.com",
				"registry": "https://mirrors.aliyun.com/pypi/simple/",
				"command": "pip"
			},
			"name": "PIP加速",
			"statusOff": true,
			"tip": "如果你没有安装pip则不需要启动它",
			"startup": {}
		},
		"overwall": {
			"targets": {
				"*.github.com": true,
				"*github*.com": true,
				"*.gitbook.io": true,
				"*.nodejs.org": true,
				"*.npmjs.com": true,
				"*.wikimedia.org": true,
				"*.v2ex.com": true,
				"*.azureedge.net": true,
				"*.cloudfront.net": true,
				"*.bing.com": true,
				"*.discourse-cdn.com": true,
				"*.gravatar.com": true,
				"*.docker.com": true,
				"*.vueuse.org": true,
				"*.elastic.co": true,
				"*.optimizely.com": true,
				"*.stackpathcdn.com": true,
				"*.fastly.net": true,
				"*.cloudflare.com": true,
				"*.233v2.com": true,
				"*.v2fly.org": true,
				"*.telegram.org": true,
				"*.amazon.com": true,
				"*.googleapis.com": true,
				"*.google-analytics.com": true,
				"*.cloudflareinsights.com": true,
				"*.intlify.dev": true,
				"*.segment.io": true,
				"*.shields.io": true,
				"*.jsdelivr.net": true,
				"*.z-library.sk": true,
				"*.zlibrary*.se": true,
				"*.discord.com": true,
				"*.wikipedia-on-ipfs.org": true,
				"*.chatgpt.com": false,
				"*.oaiusercontent.com": true,
				"*.huggingface.co": true,
				"*.pixiv.org": true,
				"*.fanbox.cc": true,
				"*.onesignal.com": true,
				"*.greasyfork.org": true,
				"*.cn-greasyfork.org": true,
				"*.notepad-plus-plus.org": true
			},
			"pac": {
				"pacFileUpdateUrl": "https://xget.xi-xu.me/gh/gfwlist/gfwlist/raw/master/gfwlist.txt",
				"enabled": true,
				"autoUpdate": true,
				"pacFilePath": "./extra/pac/pac.txt"
			},
			"name": "梯子",
			"enabled": false,
			"server": {},
			"serverDefault": {
				"ow-prod.docmirror.top": {
					"port": 443,
					"path": "X2dvX292ZXJfd2FsbF8",
					"password": "dev_sidecar_is_666"
				}
			}
		},
		"free_eye": {
			"Route": {
				"timeout": 0.1,
				"addrs": {
					"IPv4": "8.8.8.8",
					"IPv6": "2001:4860:4860::8888"
				},
				"port": 53
			},
			"DNS": {
				"timeout": 3,
				"allow": [
					"baidu.cn",
					"taobao.com",
					"www.gov.cn"
				],
				"block": [
					"wikipedia.org",
					"youtube.com",
					"facebook.com"
				]
			},
			"TCP": {
				"timeout": 3,
				"addrs": {
					"IPv4": {
						"allow": [
							"114.114.114.114",
							"223.6.6.6"
						],
						"block": [
							"8.8.8.8",
							"1.1.1.1"
						]
					},
					"IPv6": {
						"allow": [
							"2402:4e00::",
							"2400:3200:baba::1"
						],
						"block": [
							"2001:4860:4860::8888",
							"2606:4700:4700::1111"
						]
					}
				},
				"ports": [
					80,
					443
				]
			},
			"TLS": {
				"timeout": 3,
				"addrs": {
					"IPv4": "172.67.148.147",
					"IPv6": "2606:4700:3036::ac43:9493"
				},
				"snis": {
					"allow": "baidu.cn",
					"block": "wikipedia.org"
				}
			}
		},
		"node": {
			"name": "NPM加速",
			"enabled": false,
			"tip": "如果你没有安装nodejs则不需要启动它",
			"startup": {
				"variables": true
			},
			"setting": {
				"command": "npm",
				"strict-ssl": true,
				"cafile": false,
				"NODE_EXTRA_CA_CERTS": false,
				"NODE_TLS_REJECT_UNAUTHORIZED": false,
				"yarnRegistry": "default",
				"registry": "https://registry.npmjs.org"
			},
			"variables": {
				"phantomjs_cdnurl": "https://npmmirror.com/mirrors/phantomjs",
				"chromedriver_cdnurl": "https://npmmirror.com/mirrors/chromedriver",
				"sass_binary_site": "https://npmmirror.com/mirrors/node-sass",
				"ELECTRON_MIRROR": "https://npmmirror.com/mirrors/electron/",
				"NVM_NODEJS_ORG_MIRROR": "https://npmmirror.com/mirrors/node",
				"CHROMEDRIVER_CDNURL": "https://npmmirror.com/mirrors/chromedriver",
				"OPERADRIVER": "https://npmmirror.com/mirrors/operadriver",
				"ELECTRON_BUILDER_BINARIES_MIRROR": "https://npmmirror.com/mirrors/electron-builder-binaries/",
				"PYTHON_MIRROR": "https://npmmirror.com/mirrors/python"
			}
		},
		"git": {
			"name": "Git.exe代理",
			"enabled": false,
			"tip": "如果你没有安装git命令行则不需要启动它",
			"setting": {
				"sslVerify": true,
				"noProxyUrls": {
					"https://gitee.com": true,
					"https://e.coding.net": true,
					"https://codeup.aliyun.com": true
				}
			}
		}
	},
	"proxy": {
		"excludeDomesticDomainAllowList": false,
		"remoteDomesticDomainAllowListFileUrl": "https://ghproxy.net/https://raw.githubusercontent.com/pluwen/china-domain-allowlist/main/allow-list.sorl",
		"excludeIpList": {
			"objects-origin.githubusercontent.com": true,
			"*.ghproxy.net": true,
			"*.ghp.ci": true,
			"*.kkgithub.com": true,
			"*.dgithub.xyz": true,
			"pages.github.com": true,
			"help.github.com": true,
			"docs.github.com": true,
			"*.github.blog": true,
			"analytics.githubassets.com": true,
			"ghcc.githubassets.com": true,
			"www.docker.com": true,
			"login.docker.com": true,
			"api.dso.docker.com": true,
			"desktop.docker.com": true,
			"docs.docker.com": true,
			"*.s-microsoft.com": true,
			"*.xboxlive.com": true,
			"*.mihoyo.com": true,
			"*.elastic.co": true,
			"*.bilicomic.com": true,
			"[2049:8c54:813:10c::140]": true,
			"[2409:8a0c:a442:ff40:a51f:4b9c:8b41:25ea]": true,
			"[2606:2800:147:120f:30c:1ba0:fc6:265a]": true,
			"*.cmicapm.com": true,
			"*.cloudflare-cn.com": true,
			"*.microsoftonline.com": true,
			"*.msedge.net": true,
			"*kaspersky*.com": true,
			"*.upd.kaspersky.com": true,
			"*.lanhuapp.com": true,
			"*.soboten.com": true,
			"*.sandboxie-plus.com": true,
			"*.wuyou.net": true,
			"*.pyecharts.org": true,
			"*.bcloudlink.com": true,
			"*.qijishow.com": true,
			"*.z-lib.fo": true,
			"*.finalshell.com": true,
			"*.minebbs.com": true,
			"*.spigotmc.org": true,
			"*.virustotal.com": true,
			"*.gitlab.com": true,
			"*.deepseek.com": true,
			"*steaminventoryhelper.com": true,
			"*.youdemai.com": true,
			"*.casualthink.com": true,
			"44.239.165.12": true,
			"3.164.110.117": true,
			"*.cn": true,
			"*china*": true,
			"*.ghproxy.*": true,
			"ssh.github.com": true,
			"www.deepl.com": true,
			"*.csdn.net": true,
			"*.so.com": true,
			"*.baidu.com": true,
			"*.baiducontent.com": true,
			"*.bdimg.com": true,
			"*.bdstatic.com": true,
			"*.bdydns.com": true,
			"*.tencent.com": true,
			"*.qq.com": true,
			"*.weixin.com": true,
			"*.weixinbridge.com": true,
			"*.wechat.com": true,
			"*.idqqimg.com": true,
			"*.gtimg.com": true,
			"*.qpic.com": true,
			"*.qlogo.com": true,
			"*.myapp.com": true,
			"*.aliyun.com": true,
			"*.alipay.com": true,
			"*.taobao.com": true,
			"*.tmall.com": true,
			"*.alipayobjects.com": true,
			"*.dingtalk.com": true,
			"*.mmstat.com": true,
			"*.alicdn.com": true,
			"*.hdslb.com": true,
			"gitee.com": true,
			"*.gitee.com": true,
			"*.gitee.io": true,
			"*.giteeusercontent.com": true,
			"*.mozilla.org": true,
			"*.mozilla.com": true,
			"*.mozilla.net": true,
			"*.firefox.com": true,
			"*.firefox.org": true,
			"*.mozillademos.org": true,
			"*.mozillians.org": true,
			"*.mozillians.net": true,
			"*.mozillians.com": true,
			"*.sonatype.org": true,
			"*.maven.org": true,
			"*.mvnrepository.com": true,
			"*.apple.com": true,
			"*.icloud.com": true,
			"*.microsoft.com": true,
			"*.windows.com": true,
			"*.office.com": true,
			"*.office.net": true,
			"*.msn.com": true,
			"*.wps.com": true,
			"*.wps.net": true,
			"*.ksord.com": true,
			"*.qihoo.com": true,
			"*.qihucdn.com": true,
			"*.360.com": true,
			"*.360safe.com": true,
			"*.360buyimg.com": true,
			"*.360buy.com": true,
			"*.jd.com": true,
			"*.jcloud.com": true,
			"*.jcloudcs.com": true,
			"*.jcloudcache.com": true,
			"*.jcloudcdn.com": true,
			"*.jcloudlb.com": true,
			"*.bilibili.com": true,
			"*.bilivideo.com": true,
			"*.biliapi.net": true,
			"*.10086.com": true,
			"*.10086cloud.com": true,
			"*.139.com": true,
			"*.xunlei.com": true,
			"*.icpapi.com": true,
			"*.navicat.com": true,
			"localhost": true,
			"localhost.*": true,
			"127.*.*.*": true,
			"test.*": true,
			"10.*.*.*": true,
			"172.16.*.*": true,
			"172.17.*.*": true,
			"172.18.*.*": true,
			"172.19.*.*": true,
			"172.20.*.*": true,
			"172.21.*.*": true,
			"172.22.*.*": true,
			"172.23.*.*": true,
			"172.24.*.*": true,
			"172.25.*.*": true,
			"172.26.*.*": true,
			"172.27.*.*": true,
			"172.28.*.*": true,
			"172.29.*.*": true,
			"172.30.*.*": true,
			"172.31.*.*": true,
			"192.168.*.*": true
		},
		"enabled": true,
		"name": "系统代理",
		"use": "local",
		"other": [],
		"proxyHttp": false,
		"setEnv": false,
		"autoUpdateDomesticDomainAllowList": true,
		"domesticDomainAllowListFilePath": "./extra/proxy/domestic-domain-allowlist.txt"
	}
}
```
</details>

--- a/packages/gui/src/bridge/update/backend.js
+++ b/packages/gui/src/bridge/update/backend.js
@@ -18,7 +18,7 @@ const curVersion = pkg.version
 const isCurrentPreRelease = curVersion.includes('-')
 
 function extractVersion (versionData) {
-  const candidates = [versionData.name, versionData.tag_name]
+  const candidates = [versionData.tag_name, versionData.name]
   for (const candidate of candidates) {
     if (!candidate) {
       continue

